### DISCUSSION TITLE
Discrepancies in execution time with identical configurations

### DISCUSSION THREAD

**Discussion by Ding0042** [Z3Prover/z3#6810] [Q&A]
Hi, 
I have been using your Java library and noticed significant differences in execution time when running the same configuration repeatedly. 
As can be seen from the picture, conflicts and decisions are consistent, but the execution time is greatly different.

Could you shed some light on this issue? Any insights or suggestions would be greatly appreciated.
![WechatIMG161](https://github.com/Z3Prover/z3/assets/56302930/1a31c7fa-8588-4422-861c-d2fcce67e895)

**Comment by NikolajBjorner** [comment]
try running with verbose=10 and observe time taken during pre-processing tactics if they differ.
